source/component/BaseVisualization.js

```diff
@@ -1,87 +0,0 @@
-import React from 'react';
```

Comment:
So you have just moved some things to the experimental dir?

Reply:
Yes 
This folder content files that is not used in the project right now. I also added this file here as the download and home buttons doesn't use any thing from it.